### PR TITLE
Set create-schema job to restart on failure

### PR DESCRIPTION
**Pull request by jpkrohling**
See https://github.com/jaegertracing/jaeger-kubernetes/pull/43

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: incubator/jaeger/values.yaml [hunk 1]
```diff
@@ -105,10 +105,10 @@ ingress:
   # Used to create an Ingress record.
   # hosts:
   #   - chart-example.local
-  #annotations:
+  # annotations:
```
Comment (jpkrohling) [inline incubator/jaeger/values.yaml]: This (and the following) changes aren't really part of this PR, but the linter failed here.

```
Linting the Chart.yaml and values.yaml files at incubator/jaeger                                                             
incubator/jaeger/values.yaml                                                                             
  108:4     error    missing starting space in comment  (comments) 
  111:4     error    missing starting space in comment  (comments)                                       
```

**Comment by davidvonthenen** [comment]
This looks good to me!

CC: @unguiculus

**Pull request merged**